cms/static/js/factories/tag_count.js

```diff
@@ -0,0 +1,13 @@
+import * as TagCountView from 'js/views/tag_count';
+import * as TagCountModel from 'js/models/tag_count';
+
+// eslint-disable-next-line no-unused-expressions
+'use strict';
+export default function TagCountFactory(TagCountJson, el) {
+    var model = new TagCountModel(TagCountJson, {parse: true});
+    var tagCountView = new TagCountView({el, model});
+    tagCountView.setupMessageListener();
+    tagCountView.render();
+}
+
+export {TagCountFactory};
```

Comment:
Ah, these `import` and `export` statements are probably the problem - I don't think the build process used in edx-platform supports them. But it could be something else - not sure.

Reply:
There are other factories that has the [same syntax](https://github.com/openedx/edx-platform/blob/61bb17650533192ff3bca95e848142234ffb90b9/cms/static/js/factories/tag_count.js). My tutor is unconfigured, I'm going to try to configure it to try another syntax

Reply:
That error happened to me, but it was fixed with [this line](https://github.com/openedx/edx-platform/pull/34059/files#diff-fbed72c8f21e30539a12e9e1dfdff135c75340e4ca10894890d1a21c6a0988d5R87). Maybe clearing the cache?

Reply:
Thx, let me know when you've found a fix.

Reply:
@bradenmacdonald I fixed the bug. It was an extrra `,` [here](https://github.com/openedx/edx-platform/pull/34059/commits/6b78dc94247f89a7d850b3d80088d6a42a70938b#diff-ba366278f32578c3c9ad5b048f4d212a668e20c4abaf89eed318593b1006a600L36) You can test it? I still need to configure my tutor with the MFE